docs/developers-guide/e2e-tests.md

```diff
@@ -149,9 +149,24 @@ yarn test-cypress-open-qa
 
 Tests that depend on Snowplow expect a running server. To run them, you need to:
 
-- run Snowplow locally: `docker-compose -f ./snowplow/docker-compose.yml up -d`
 - pass env variables to the test run: `MB_SNOWPLOW_AVAILABLE=true MB_SNOWPLOW_URL=http://localhost:9090 yarn test-cypress-open`
 
+## Testing with Snowplow
+Our end-to-end testing environment has been configured to run Snowplow Micro alongside the application.
+
+To run Snowplow locally use the following commands:
+
+```
+docker-compose -f ./snowplow/docker-compose.yml up -d
+MB_SNOWPLOW_AVAILABLE=true
+MB_SNOWPLOW_URL=http://localhost:9090
```

Comment:
if I understand correctly, you need to export as well

```suggestion
export MB_SNOWPLOW_AVAILABLE=true
export MB_SNOWPLOW_URL=http://localhost:9090
```

Reply:
I guess depending on how you run it, but it’s true that in use case shown, `export` is needed. will fix